File: testdata/ChangeFieldType/Cpp/current/Temperature.cpp
#include <string>
#include <iostream>

class Temperature {
private:
    std::string value; // Temperature in Celsius as a String

public:
    Temperature(const std::string& value) : value(value) {}

    std::string getValue() const {
        return value;
    }

    void setValue(const std::string& value) {
        this->value = value;
    }

    std::string toString() const {
        return "Temperature: " + value + "°C";
    }
};

File: testdata/ChangeFieldType/Java/current/Temperature.java
public class Temperature {
    private String value; // Temperature in Celsius as a String

    public Temperature(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    public String toString() {
        return "Temperature: " + value + "°C";
    }
}
